Classes/Wwwision/Neos/MailChimp/Domain/Service/MailChimpService.php

```diff
@@ -97,10 +97,15 @@ public function getMembersByListId(string $listId)
     {
         $memberQuery = new CallbackQuery(function (CallbackQuery $query) use ($listId) {
             $members = $this->get("lists/$listId/members", ['offset' => $query->getOffset(), 'count' => $query->getLimit()]);
-            return $members['members'];
+            return array_filter(
+                $members['members'],
+                static function (array $member) {
+                    return $member['status'] === 'subscribed';
+                }
+            );
         }, function () use ($listId) {
-            $members = $this->get("lists/$listId/members", ['count' => 0]);
```

Comment:
Fun fact: setting count to 0 still returned all entries… the best "solution" would have been a count of 1 for this approach. 🤷‍♂️